app/src/main/java/com/github/se/polyfit/ui/screen/recipeRec/RecipeRecommendationScreen.kt

```diff
@@ -58,15 +62,31 @@ fun recipeDisplay(
               fontWeight = FontWeight.Bold,
```

Comment:
```suggestion
              fontWeight = FontWeight.Bold,
              modifier = Modifier.padding(8.dp)
```
I'd suggest adding some padding between the side of the screen and the text, just to make it visually more appealing. I'd also suggest using another colour like PurpleGrey to make the colour less harsh and more inline with the theming